addons/account/i18n/vi.po

```diff
@@ -13607,7 +13607,7 @@ msgstr "Chuyển cho %s"
 #. module: account
 #: model_terms:ir.ui.view,arch_db:account.view_account_payment_search
 msgid "Transfers"
-msgstr "Chuyển khoản"
+msgstr "Giao dịch nội bộ"
```

Comment:
Xem kỹ lại xem có phải thuật ngữ Transfer chỉ dùng cho chuyển tiền nội bộ hay không.

Reply:
vâng ạ

Reply:
Em có kiểm tra lại thì tính năng của nó chỉ lọc ra những `Thanh toán` đánh dấu là `Là giao dịch nội bộ` thôi ạ
- code bộ lọc `<filter string="Transfers" name="transfers_filter" domain="[('is_internal_transfer', '=', True)]"/>`

CC: @jennie-pham